pycbc/io/live.py

```diff
@@ -281,8 +303,6 @@ def upload(self, fname, gracedb_server=None, testing=True,
             test trigger (True) or a production trigger (False).
         """
         from ligo.gracedb.rest import GraceDb
-        import matplotlib
-        matplotlib.use('Agg')
```

Comment:
I think it is a good idea to keep these two lines, as PyCBC Live runs on a headless system.

Reply:
Oh, I don't know why I removed them, sorry!